README.md

```diff
@@ -21,9 +21,9 @@ By the end of this tutorial you will be able to create a simple HTML front end w
 
    - Change to the Ropsten Tesnet and get a Copy an account's wallet public address
 
-2. **Request some Ropsten Tesnet Ether from a faucet loaded into your Metamask Wallet.**
```

Comment:
Instead of removing Ropsten, can you specify that they can also use Rinkeby? I do not wanna delete Ropsten

Reply:
Yeah, you are right sure.